### PR TITLE
Issue #127

### PR DESCRIPTION
**Pull request by fadlinurhasan**

### CODE REVIEW

File: app/controllers/app_groups_controller.rb [hunk 1]
```diff
@@ -3,8 +3,21 @@ class AppGroupsController < ApplicationController
   before_action :set_app_group, only: %i(show update manage_access)
 
   def index
-    @app_groups = policy_scope(AppGroup).order(:name)
     @allow_create_app_group = policy(AppGroup).new?
+    # app_groups = policy_scope(AppGroup)
```
Comment (poedja) [inline app/controllers/app_groups_controller.rb]: Please remove unnecessary comment

File: app/models/app_group.rb [hunk 2]
```diff
@@ -12,6 +12,52 @@ class AppGroup < ApplicationRecord
...
+              :available_filters => %w[
+                sorted_by
+                search_query
+              ]
+
+  scope :search_query, ->(query) {
+    return nil  if query.blank?
+    # condition query, parse into individual keywords
+    terms = query.downcase.split(/\s+/)
+    # replace "*" with "%" for wildcard searches,
```
Comment (poedja) [inline app/models/app_group.rb]: Please remove these unnecessary comments

File: app/models/infrastructure.rb [hunk 3]
```diff
@@ -143,4 +143,10 @@ def component_hash(type, count)
     name = "#{self.cluster_name}-#{type}-#{format('%02d', count.to_i)}"
     { name: name, type: type}
   end
+
+  def self.options_for_select
+    infrastructures = Infrastructure.arel_table
+    # order('LOWER(name)').map { |e| [e.name, e.id] }
```
Comment (poedja) [inline app/models/infrastructure.rb]: Please remove this unnecessary comment

File: app/views/layouts/application.html.slim [hunk 4]
```diff
@@ -5,9 +5,9 @@ html
     link [rel='stylesheet' href='https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css' integrity='sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4' crossorigin='anonymous']
     link [rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous"]
     = stylesheet_link_tag 'https://gitcdn.github.io/bootstrap-toggle/2.2.2/css/bootstrap-toggle.min.css'
-    = stylesheet_link_tag 'application', media: 'all'
+    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
```
Comment (poedja) [inline app/views/layouts/application.html.slim]: Please remove this since it is unused

File: config/initializers/kaminari_config.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+Kaminari.configure do |config|
+  config.default_per_page = 10
```
Comment (poedja) [inline config/initializers/kaminari_config.rb]: Please move this value as environment variable

File: app/controllers/app_groups_controller.rb (hunk 1 above)
Comment (fadlinurhasan) [reply app/controllers/app_groups_controller.rb]: Done. https://github.com/BaritoLog/BaritoMarket/pull/134/commits/415056c98c6b193a5d1a3f996e160d268406725e

File: app/models/app_group.rb (hunk 2 above)
Comment (fadlinurhasan) [reply app/models/app_group.rb]: Done https://github.com/BaritoLog/BaritoMarket/pull/134/commits/415056c98c6b193a5d1a3f996e160d268406725e

File: app/models/infrastructure.rb (hunk 3 above)
Comment (fadlinurhasan) [reply app/models/infrastructure.rb]: Done https://github.com/BaritoLog/BaritoMarket/pull/134/commits/415056c98c6b193a5d1a3f996e160d268406725e

File: app/views/layouts/application.html.slim (hunk 4 above)
Comment (fadlinurhasan) [reply app/views/layouts/application.html.slim]: Done. https://github.com/BaritoLog/BaritoMarket/pull/134/commits/cf4fb880ced907997651bca3739431fcb8adec43

File: config/initializers/kaminari_config.rb (hunk 5 above)
Comment (fadlinurhasan) [reply config/initializers/kaminari_config.rb]: Done. https://github.com/BaritoLog/BaritoMarket/pull/134/commits/9a39ff7f03b901ed75307d63abe945ab62079347

**Pull request merged**